### PR TITLE
Clean groupby error message

### PR DESCRIPTION
**Pull request by dsaxton**
Cleans an error message from `groupby.ops`.

### CODE REVIEW

File: pandas/core/groupby/ops.py [hunk 1]
```diff
@@ -464,7 +464,7 @@ def _cython_operation(self, kind, values, how, axis, min_count=-1, **kwargs):
         # are not setup for dim transforming
         if is_categorical_dtype(values) or is_sparse(values):
             raise NotImplementedError(
-                "{} are not support in cython ops".format(values.dtype)
+                "{} are not supported in Cython operations".format(values.dtype)
```
Comment (jbrockmendel) [inline pandas/core/groupby/ops.py]: Would this be clearer as "{dtype} dtype is not supported [...]"?

Comment (dsaxton) [reply pandas/core/groupby/ops.py]: I think so, feels like error messages shouldn't expose implementation details like this

Comment (jbrockmendel) [reply pandas/core/groupby/ops.py]: yah, the groupby code involves a lot of nested try/except catching, so i dont know off the top of my head whether this gets expose to the user.

Regardless, small improvements are still improvements, and they're easy to review.

**Comment by alimcmaster1** [comment]
Just need to format this with black:

```
Checking black formatting
would reformat /home/vsts/work/1/s/pandas/core/groupby/ops.py
1 file would be reformatted, 870 files would be left unchanged.
```

Thanks!

**Comment by jreback** [comment]
this is ok, though this is not actually a user facing error message.

**Pull request merged**